torch/csrc/autograd/python_function.h

```diff
@@ -29,6 +29,10 @@ struct PyFunction : public Function {
   PyObject* obj;
 };
 
+using input_map = std::unordered_map<InputNode*, std::shared_ptr<Variable>>;
+using output_list = std::vector<Output>;
```

Comment:
it'd be better to have a single definition for this. Otherwise they can get out of sync and be really confusing